public/js/user.js

```diff
@@ -450,5 +539,21 @@ function contentEditwindow(site) {
   html += "</select></div>";
   html += "<input type='hidden' id='site' value='"+site.id+"'/>";
   html += "<div class=''><button type='submit' class='btn btn-primary'>Save</button></div></div></form></div></div>";
+
+  if((site.created_by === currentUser || isAdmin) && isValidMarkerId(site.id)) {
+    html += "<div id='adv-button'><span class='glyphicon glyphicon glyphicon-cog'></span> Advanced Options &nbsp; <span class='glyphicon glyphicon-chevron-down' id='adv-icon'></span></div>"
+    html += "<div id='adv-options'>"
+    
+    html += "<div class='form-inline'><br>Co-owner"
```

Comment:
Instead of a separate label, make "Co-owner" a placeholder for the input field. This will allow the input field to be wider and accommodate most usernames without truncation.

Reply:
I still had to truncate it, as the input was pushing the button to the next line :/